.github/workflows/triage.yml

```diff
@@ -16,6 +16,9 @@ jobs:
   triage:
     name: Nissuer
     runs-on: ubuntu-latest
+    if: |
```

Comment:
I could be wrong but I think you need to do something like [this](https://github.com/vercel/next.js/blob/cdb275c7b8d77afc27026584e2b0c15448d9688c/.github/workflows/retry_deploy_test.yml#L19) for a multi-line expression